### DISCUSSION TITLE
[4.0] Preview Module Positions link in Module View

### DISCUSSION THREAD

**Discussion by caseyzman** [joomla/joomla-cms#23067] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
UX problem

### Describe the solution you'd like
When editing in the Modules View in the admin panel, including a link similar to the "Preview Module Positions" for the default template positions would be helpful. (see pic below). The link should only show when the option is enabled in the template options.

It would also be helpful to have this preview link in the "Menus: Edit Item" view "Module Assignment" tab.

### Additional context
![module-position-preview](https://user-images.githubusercontent.com/4513781/48431097-7c08c580-e736-11e8-8dc2-e8a20a1f6edc.jpg)

**Comment by ghost** [comment]
@wilsonge is this an Issue which should go to j4?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23067">issues.joomla.org/tracker/joomla-cms/23067</a>.</sub>

**Comment by ghost** [comment]
@wilsonge reminder.

**Comment by infograf768** [comment]
This is not an issue, it is asking for a new feature.
Therefore it depends on the will of someone to code it, _if feasible_ as it looks quite complex, specially when using multiple templates.

**Comment by wilsonge** [comment]
What JM says. It's definitely a cool concept. But not high on my priority list at this time
